src/libcore/ptr.rs

```diff
@@ -212,6 +212,14 @@ pub unsafe fn write_volatile<T>(dst: *mut T, src: T) {
 #[lang = "const_ptr"]
 impl<T: ?Sized> *const T {
     /// Returns true if the pointer is null.
+    ///
+    /// Example:
+    ///
```

Comment:
Ooooooh. Isn't it a little overloaded to have "Example" and "basic usage" for just one example?